includes/dbFacile.php

```diff
@@ -109,6 +109,63 @@ function dbInsert($data, $table) {
 
 
 /*
+ * Passed an array and a table name, it attempts to insert the data into the table.
+ * $data is an array (rows) of key value pairs.  keys are fields.  Rows need to have same fields.
+ * Check for boolean false to determine whether insert failed
+ * */
+
+
+function dbBulkInsert($data, $table) {
+    global $fullSql;
+    global $db_stats;
+    // the following block swaps the parameters if they were given in the wrong order.
+    // it allows the method to work for those that would rather it (or expect it to)
+    // follow closer with SQL convention:
+    // insert into the TABLE this DATA
+    if (is_string($data) && is_array($table)) {
+        $tmp   = $data;
+        $data  = $table;
+        $table = $tmp;
+        // trigger_error('QDB - Parameters passed to insert() were in reverse order, but it has been allowed', E_USER_NOTICE);
```

Comment:
I know this part is copy/paste from `dbInsert()` but I think we can skip this line here to avoid more useless duplication ;)